servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.cpp

```diff
@@ -41,8 +41,12 @@ Transform3D RenderSceneDataRD::get_cam_transform() const {
 	return cam_transform;
 }
 
-Projection RenderSceneDataRD::get_cam_projection() const {
-	return cam_projection;
+Projection RenderSceneDataRD::get_cam_projection(bool p_flip_y) const {
```

Comment:
Is there any way that we could detect if the RenderSceneData is being used in ReflectionProbes? I would really like to remove the p_flip_y option if we can. 

For starters, it will always be false in the GLES3 version of this class. So the user shouldn't have to think about that. 

Perhaps we could store whether a ReflectionProbe is being used when we create the class?

Alternatively, we could overhaul the renderer so we aren't rendering upside down. As far as I can tell there has never been a benefit to rendering upside down.

Reply:
Just to clue everyone else in on our discussion this morning.

Our decision for now is that for 4.3 we'll try and make the logic in RenderSceneData* automatic, I'll look into the needed changes to make this possible and update this PR.

Than for 4.4 we're going to investigate removing the flip logic all together. I'll raise a proposal with the details for this.

Reply:
If we could just render right side up always, in both RD and compatibility, that would be amazing! :-)